File: Week_1/problem_2-print_math.py
# Input two numbers
a, b = map(int, input().split())

# Perform operations
print(f"{a} + {b} = {a + b}")
print(f"{a} - {b} = {a - b}")
print(f"{a} * {b} = {a * b}")

# Set precision only for division
print(f"{a} / {b} = {a / b:.2f}")

File: Week_1/problem_2-print_math.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    double a, b;
    cin >> a >> b;

    cout << a << " + " << b << " = " << a+b << endl;
    cout << a << " - " << b << " = " << a-b << endl;
    cout << a << " * " << b << " = " << a*b << endl;

    // Set precision only for division
    cout << fixed << setprecision(2); // Set precision to 2
    cout << a << " / " << b << " = " << a / b << endl;

    return 0;
} 